code/modules/projectiles/projectile/energy/nuclear_particle.dm

```diff
@@ -52,7 +52,7 @@
 			name = "impossibly strong nuclear particle"
 			damage = 30
 
-/atom/proc/fire_nuclear_particle(angle = rand(0,360), customize = FALSE, custompower = 1e12) //used by fusion to fire random nuclear particles. Fires one particle in a random direction.
+/atom/proc/fire_nuclear_particle(angle = rand(0,360), customize = FALSE, custompower = INFINITY) //used by fusion to fire random nuclear particles. Fires one particle in a random direction.
```

Comment:
```suggestion
/atom/proc/fire_nuclear_particle(angle = rand(0,360), customize = FALSE, custompower = 1e12) //used by fusion to fire random nuclear particles. Fires one particle in a random direction.
```

Reply:
Just don't bother this is probably beeing overwriten by the proccal anyways so lets just apply infinity down the procchain instead